src/app/base/components/NodeSummaryNetworkCard/_index.scss

```diff
@@ -0,0 +1,6 @@
+@mixin NodeSummaryNetworkCard {
+    .network-card {
+        padding: 0.5rem 1rem 1rem 1rem;
```

Comment:
Typically we want to avoid using specific spacing numbers so that if we ever change the design the spacing can be adjusted by changing the variables. This could be rewritten using the [spacing variables](https://vanillaframework.io/docs/settings/spacing-settings) to `padding: $spv--small $sph--large $spv--large $sph--large`,